src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -43,7 +49,8 @@ private WebDriverConfig() {
             implicit = properties.getProperty("implicit");
             explicit = properties.getProperty("explicit");
         } catch (Exception ex) {
-            ex.printStackTrace();
+            Log.getInstance().getLog().error(ex);
+            throw new NullPointerException("File not found" + ex);
         }
     }
 
```

Comment:
write again your documentation, must be written in third person

Reply:
That is write in third person.